plasoscaffolder/scaffolders/__init__.py

```diff
@@ -1,3 +1,5 @@
 # -*- coding: utf-8 -*-
 """This file imports Python modules that registers scaffolders."""
 from plasoscaffolder.scaffolders import plaso_sqlite
+from plasoscaffolder.scaffolders import timesketch_index
```

Comment:
Looks like these changes leaked into the PR - please remove

Reply:
I merged master updates into the PR, for testing purposes. Still want me to remove these? this is just merging the changes in from master